doc/python/b64.md

```diff
@@ -0,0 +1,198 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.15.2
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.9.0
+  plotly:
+    description: How to format axes of 3d plots in Python with Plotly.
+    display_as: b64
+    language: python
+    layout: base
+    name: b64
+    order: 1
+    page_type: example_index
+    permalink: python/b64/
+    thumbnail: thumbnail/b64.png
+---
+
+### Simple example showing how arrays of numbers could be passed as base64 typed array objects to plotly.js
+
+```python
+import plotly.graph_objects as go
+
+# x = [-200000 -100000       0  100000  200000]
+x = {'dtype': 'int32', 'bdata': 'wPL8/2B5/v8AAAAAoIYBAEANAwA='}
+
+# y = [0 1 2 3 4 5 6 7 8 9]
+y = {'dtype': 'uint8', 'bdata': 'AAECAwQFBgcICQ=='}
+
+# z = [
+# [  61 -295 -765  863  932]
+# [-897   96  724  791 -993]
+# [ -95 -796 -285  381  669]
+# [ 985 -153  425  -40  136]
+# [-856  955 -871  414  996]
+# [ 966  607 -154 -251 -882]
+# [-492 -116  414  426  305]
+# [ 919  202 -505  300 -833]
+# [ 278 -152 -643 -950  -86]
+# [ 898 -532  608  -93  110]]
+z = {
+  'dtype': 'int16',
+  'bdata': 'PQDZ/gP9XwOkA3/8YADUAhcDH/yh/+T84/59AZ0C2QNn/6kB2P+IAKj8uwOZ/J4B5APGA18CZv8F/478FP6M/54BqgExAZcDygAH/iwBv/wWAWj/ff1K/Kr/ggPs/WACo/9uAA==', 'shape': '10, 5'
+}
+
+fig = go.Figure(data=[go.Surface(
+    x=x,
+    y=y,
+    z=z
+)])
+
+fig.show()
+```
+
+### Example where base64 is applied to pass values as typed array objects to plotly.js
+
+```python
+import plotly.graph_objects as go
```

Comment:
```suggestion
### Example where base64 is applied to pass values as typed array objects to plotly.js

```python
import plotly.graph_objects as go
```
If we'll do this automatically, can this example go?

Reply:
I prefer to keep it for now.